### DISCUSSION TITLE
pandas/python incompatibilty

### DISCUSSION THREAD

**Discussion by r00t-64** [OpenBB-finance/OpenBB#6493] [General]
Sorry if this a dumb question, but i recently installed openbb on linux and got this:

2024 Jun 09, 17:27  (🦋) /equity/ $ profile --symbol aapl

/home/root64/miniconda3/lib/python3.11/site-packages/openbb_cli/controllers/utils.py:386: FutureWarning: errors='ignore' is deprecated and will raise in a future version. Use to_numeric without passing `errors` and catch exceptions explicitly instead
  df[col] = pd.to_numeric(df[col], errors="ignore")
/home/root64/miniconda3/lib/python3.11/site-packages/openbb_cli/controllers/utils.py:416: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  df_outgoing = df_outgoing.applymap(str)

**Comment by gchan510** [comment]
Same issue here running with pandas 2.2.2. Looks like FutureWarning:error='ignore' was changed in version 2.2 and DataFrame.applymap() deprecated in version 2.1.0
